packages/eas-cli/src/metadata/apple/config/writer.ts

```diff
@@ -35,7 +55,7 @@ export class AppleConfigWriter {
 
     this.schema.info[attributes.locale] = {
       ...existing,
-      title: attributes.name ?? 'no name provided',
+      title: attributes.name ?? 'App title', // see schema #/definitions/apple/AppleInfo/defaultSnippets
```

Comment:
This might fail if "App title" is taken so really we'd just be delaying the failure until later.

Reply:
We can also opt to remove it, I added the default because store config errors were unskippable

Reply:
done